### ISSUE TITLE
Tailwind doesn't support css variables in screen options

### ISSUE THREAD

**Issue by thomaslecoeur** [songkeys/tailwind-preset-mantine#2]
Just noticed that tailwind screen otpions doesn't support css variables (because of browser limitations). So classes such as `md:grid-cols:3` do not work with current preset implementation.

```
screens: {
          xs: 'var(--mantine-breakpoint-xs)',
          sm: 'var(--mantine-breakpoint-sm)',
          md: 'var(--mantine-breakpoint-md)',
          lg: 'var(--mantine-breakpoint-lg)',
          xl: 'var(--mantine-breakpoint-xl)',
        },
```

Maybe the best would be to pass the whole theme to the preset to be able to get the variables values directly instead of the css vars?

**Comment by songkeys** [comment]
Nice catch! You're right. I just noticed this 🤔 https://github.com/tailwindlabs/tailwindcss/issues/1233

In the current API design, users can only pass `mantineColors` from `mantineTheme.colors`. I believe we can introduce a `mantineBreakpoints` property to allow passing `mantineTheme.breakpoints` as well.

The reason I don’t allow users to pass the entire `mantineTheme` configuration is that [`mantineTheme.components`](https://mantine.dev/theming/theme-object/#components) might include (s)css modules, which could fail to resolve due to the absence of an (s)css loader when loading the Tailwind config file. Therefore, I aim to limit the properties as much as possible. I'm not sure if there's a better approach.

What do you think? Would you like to send a pull request to address this issue?

**Comment by thomaslecoeur** [comment]
For now did a PR with entire theme. Maybe we can pass only breakpoints if you think passing the whole theme is not a good idea.

https://github.com/songkeys/tailwind-preset-mantine/pull/3

**Comment by songkeys** [comment]
Thank you. Maybe going for only the `breakpoints` property is better. In the next major version, once I figure out how to solve the css issue, we can pass the whole config.

**Comment by songkeys** [comment]
Closed via #3
